.ci/openshift-ci/images/create_links.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat Inc.
```

Comment:
Just wanted to check that the two copyright dates on this PR are correct?

Reply:
Guest what? Copy-and-paste when you rushing with too many things... :)
I will fix it.